source/fundamentals/indexes.txt

```diff
@@ -234,17 +248,18 @@ Unique indexes ensure that the indexed fields do not store duplicate values. By
 on the ``_id`` field during the creation of a collection. To create a unique index, specify the field or combination of
 fields that you want to prevent duplication on and set the ``unique`` option to ``true``.
 
-The following example uses the ``createIndex()`` method to create a unique, descending index on the ``theaterId`` field in the
-``theaters`` collection of the ``sample_mflix`` database.
+The following example creates a unique, descending index on the ``theaterId`` field:
 
 .. literalinclude:: /includes/fundamentals/code-snippets/IndexPage.java
    :language: java
    :dedent:
    :start-after: begin unique index
    :end-before: end unique index
 
-If you attempt to perform a write operation that stores a duplicate value that violates the unique index, the MongoDB
-Java driver will raise a ``DuplicateKeyException``, and MongoDB will throw an error resembling the following:
+.. warning::
+
+   If you attempt to perform a write operation that stores a duplicate value that violates the unique index, the MongoDB
+   Java driver will raise a ``DuplicateKeyException``, and MongoDB will throw an error resembling the following:
```

Comment:
Really liking the "footguns in notes" pattern here that you've established for each section.